packages/battery_plus/tizen/src/battery_plus_tizen_plugin.cc

```diff
@@ -193,6 +158,43 @@ class BatteryPlusTizenPlugin : public flutter::Plugin {
     result->Success(flutter::EncodableValue(percentage));
   }
 
+ private:
+  void SetupChannels(flutter::PluginRegistrar *registrar) {
+    auto method_channel =
+        std::make_unique<flutter::MethodChannel<flutter::EncodableValue>>(
+            registrar->messenger(), "dev.fluttercommunity.plus/battery",
+            &flutter::StandardMethodCodec::GetInstance());
+    m_event_channel =
```

Comment:
Could you change prefix ```m_``` to postfix ```_``` to represent member of class?
I think this suggestion is out of scope, but I'd appreciate it if you could

Reply:
IMO, renaming the variables with a new style is quite a big change which should be done in a separate commit. I still hope someone to do the work for all our packages.

Reply:
@bbrto21, I didn't want to mix `m_` and `_` in one file, so I decided to use `m_` in part of the plugins. I don't like it either :)